<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveBroadcastServiceClass</class>
 <widget class="QWidget" name="LiveBroadcastServiceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>960</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1012</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LiveBroadcastService</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>15</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>LOOK直播</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(102, 102, 102);</string>
   </property>
   <property name="text">
    <string>在这里，看见音乐</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>922</x>
     <y>30</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	font: 12pt &quot;黑体&quot;;
	font-weight:bold;
	color: rgb(102, 102, 102);
}
QPushButton{
	font: 12pt &quot;黑体&quot;;
	color: rgb(102, 102, 102);
}
QPushButton:checked{
	font: 12pt &quot;黑体&quot;;
	color: rgb(102, 102, 102);
}</string>
   </property>
   <property name="text">
    <string>更多 &gt;</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>310</y>
     <width>222</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>61</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>佛系聊天室最开心啦</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>310</y>
     <width>222</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>61</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>首次加团送半首歌</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>310</y>
     <width>222</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>61</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>夜幕男神 招战神</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>753</x>
     <y>310</y>
     <width>222</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>61</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>全麦优质视频</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>620</y>
     <width>222</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>61</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>欢迎加入奇妙公馆一起玩</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>620</y>
     <width>222</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>61</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>天呐！这房间也太上头了</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>753</x>
     <y>620</y>
     <width>222</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>61</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>点唱新厅试营业</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>620</y>
     <width>222</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>61</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>你的女友等你回家</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>80</y>
     <width>222</width>
     <height>222</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/直播/1.png</normaloff>../../res/直播/1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>80</y>
     <width>222</width>
     <height>222</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/直播/2.png</normaloff>../../res/直播/2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>80</y>
     <width>222</width>
     <height>222</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/直播/3.png</normaloff>../../res/直播/3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>753</x>
     <y>80</y>
     <width>222</width>
     <height>222</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/直播/4.png</normaloff>../../res/直播/4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>390</y>
     <width>222</width>
     <height>222</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/直播/5.png</normaloff>../../res/直播/5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>390</y>
     <width>222</width>
     <height>222</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/直播/6.png</normaloff>../../res/直播/6.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>390</y>
     <width>222</width>
     <height>222</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/直播/7.png</normaloff>../../res/直播/7.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>753</x>
     <y>390</y>
     <width>222</width>
     <height>222</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/直播/8.png</normaloff>../../res/直播/8.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>222</width>
     <height>222</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
